docs/reference/nanodet.md

```diff
@@ -0,0 +1,289 @@
+  "t", "g", "m", "m-416", "m-0.5x", "m-1.5x", "m-1.5x-416", "plus-m_320", "plus-m-1.5x_320", "plus-m_416",
+  "plus-m-1.5x_416", "custom"}*\
+  Specifies the model to use and the config file that contains all hyperparameters for training, evaluation and inference as the original
+  [Nanodet implementation](https://github.com/RangiLyu/nanodet). If you want to overwrite some of the parameters you can
+  put them as parameters in the learner.
+- **iters**: *int, default=None*\
+  Specifies the number of epochs the training should run for.
+- **lr**: *float, default=None*\
+  Specifies the initial learning rate to be used during training.
+- **batch_size**: *int, default=None*\
+  Specifies number of images to be bundled up in a batch during training.
+  This heavily affects memory usage, adjust according to your system.
+- **checkpoint_after_iter**: *int, default=None*\
+  Specifies per how many training iterations a checkpoint should be saved.
+  If it is set to 0 no checkpoints will be saved.
+- **checkpoint_load_iter**: *int, default=None*\
+  Specifies which checkpoint should be loaded.
+  If it is set to 0, no checkpoints will be loaded.
+- **temp_path**: *str, default='temp'*\
+  Specifies a path where the algorithm looks for saving the checkpoints along with the logging files.
+- **device**: *{'cpu', 'cuda'}, default='cuda'*\
+  Specifies the device to be used.
+- **weight_decay**: *float, default=None*\
+- **warmup_steps**: *int, default=None*\
+- **warmup_ratio**: *float, default=None*\
+- **lr_schedule_T_max**: *int, default=None*\
+- **lr_schedule_eta_min**: *float, default=None*\
+- **grad_clip**: *int, default=None*\
+
+#### `NanodetLearner.fit`
+```python
+NanodetLearner.fit(self, dataset, val_dataset, logging_path, verbose, local_rank, seed)
+```
+
+This method is used for training the algorithm on a train dataset and validating on a val dataset.
+
+Parameters:
+
+- **dataset**: *ExternalDataset*\
+  Object that holds the training dataset.
+  Can be of type `ExternalDataset`.
+- **val_dataset** : *ExternalDataset, default=None*\
+  Object that holds the validation dataset.
+  Can be of type `ExternalDataset`.
+- **logging_path** : *str, default=''*\
+  Subdirectory in temp_path to save log files and TensorBoard.
+- **verbose** : *bool, default=True*\
+  Enables the maximum verbosity and the logger.
+- **seed** : *int, default=123*\
+  Seed for repeatability.
+
+#### `NanodetLearner.eval`
+```python
+NanodetLearner.eval(self, dataset, verbose)
+```
+
+This method is used to evaluate a trained model on an evaluation dataset.
+Saves a txt logger file containing stats regarding evaluation.
+
+Parameters:
+
+- **dataset** : *ExternalDataset*\
+  Object that holds the evaluation dataset.
+- **verbose**: *bool, default=True*\
+  Enables the maximum verbosity and logger.
+
+#### `NanodetLearner.infer`
+```python
+NanodetLearner.infer(self, input, thershold)
```

Comment:
it seems the `verbose` argument documentation is missing

Reply:
Thank you for your comment.